### PR TITLE
Fix decals not deserializing

### PR DESCRIPTION
**Pull request by ElectroJr**
Bug added by my recent decal PR. Just tested loading, editing, saving, and then re loading the map and everything seems fine?

:cl:
- fix: Fix tile decals not loading properly.

### CODE REVIEW

File: Content.Shared/Decals/DecalGridComponent.cs [hunk 1]
```diff
@@ -23,7 +24,7 @@ public sealed class DecalGridComponent : Component
         [Serializable, NetSerializable]
         public sealed class DecalChunk
         {
-            [DataField("decals")]
+            [IncludeDataField(customTypeSerializer:typeof(DictionarySerializer<uint, Decal>))]
             public Dictionary<uint, Decal> Decals;
```
Comment (ElectroJr) [inline Content.Shared/Decals/DecalGridComponent.cs:28]: I don't fully understand why its necessary, but it seems to error if the serializer is not explicitly specified?

Comment (ElectroJr) [reply Content.Shared/Decals/DecalGridComponent.cs:28]: If I'm reading the code in `SerializationManager` correctly, its intentional that this only works without a custom type serializer if the type is a data definition? Is what I'm doing here cursed for some reason?

Comment (PaulRitter) [reply Content.Shared/Decals/DecalGridComponent.cs:28]: it should work without the customtypeserializer too. the checks at init just ensure that we dont have a recursion in includedatafields

Comment (ElectroJr) [reply Content.Shared/Decals/DecalGridComponent.cs:28]: Well without it `SerializationManager` throws a "Invalid Types used for include fields" and afaict its not related to a recursion check

**Pull request merged**